README.md
<p align="center">
  <a href="https://www.nordtheme.com" target="_blank">
    <picture>
      <source srcset="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/logos/heroes/logo-typography/dark/frostic/nord3/spaced.svg?sanitize=true" width="100%" media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)" />
      <source srcset="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/logos/heroes/logo-typography/light/frostic/nord6/spaced.svg?sanitize=true" width="100%" media="(prefers-color-scheme: dark)" />
      <img src="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/logos/heroes/logo-typography/dark/frostic/nord3/spaced.svg?sanitize=true" width="100%" />
    </picture>
  </a>
</p>

<p align="center">
  <a href="https://github.com/nordtheme/java/releases/latest" target="_blank">
    <img src="https://img.shields.io/github/release/nordtheme/assets.svg?style=flat-square&label=Release&logo=github&logoColor=eceff4&colorA=4c566a&colorB=88c0d0" />
  </a>
  <a href="https://github.com/nordtheme/java/blob/main/changelog.md" target="_blank">
    <img src="https://img.shields.io/github/release/nordtheme/assets.svg?style=flat-square&label=Changelog&logo=github&logoColor=eceff4&colorA=4c566a&colorB=88c0d0" />
  </a>
</p>

<p align="center">
  Official <a href="https://www.java.com" target="_blank" rel="noreferrer">Java</a> implementation of <a href="https://www.nordtheme.com" target="_blank">Nord</a> — An arctic, north-bluish color palette.
</p>

This repository is currently in a transitioning phase and work in progress!

<p align="center">
  <picture>
    <source srcset="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/elements/separators/iceberg/footer/dark/spaced.svg?sanitize=true" width="100%" media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)" />
    <source srcset="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/elements/separators/iceberg/footer/light/spaced.svg?sanitize=true" width="100%" media="(prefers-color-scheme: dark)" />
    <img src="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/elements/separators/iceberg/footer/dark/spaced.svg?sanitize=true" width="100%" />
  </picture>
</p>

<p align="center">
  Copyright &copy; 2016-present <a href="https://www.svengreb.de" target="_blank">Sven Greb</a>
</p>

<p align="center">
  <a href="https://github.com/nordtheme/java/blob/main/license" target="_blank">
    <img src="https://img.shields.io/static/v1.svg?style=flat-square&label=License&message=MIT&logoColor=eceff4&logo=creativecommons&colorA=4c566a&colorB=88c0d0"/>
  </a>
  <a href="https://www.svengreb.de">
    <img src="https://img.shields.io/static/v1.svg?style=flat-square&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAABmJLR0QA/wD/AP+gvaeTAAABMklEQVQ4jcWQvUoDQRRGz52s5IfVIiDWPkGKFFaCIVaGdIagjcFAwICFb7DvIK6QQlNpY2UQLMQVBbEQ0SewFkGbKCQmOzaTJay7/lR+zTAf9xwuF/47Mv45rdezqWEq72v/RWZnHgqOMwDwHMfSj085JSqb6Pu38we7r18E3nqzhmYbsE11rxKsAvhDfQiSM30XYbOw57YDwfnaRl6U3ABWaMNn806H+oGPzBX3d+4UgChZiYBHYBgGsBLoKoAyhR0x9G20Zmpc4P1ZoMQDcwMNclFrdhBKv6M5WWi7ZQGtjEUn35IV4OwnVjSX/WGmKqCDDUa5rmyle3bvGFiMg3WGUsF1u0EXHoqTRMGRgkAy2eugKZrqijRLYThWANBpNDL2h3UE0J0YLJdbrfe42f/NJ0wqY7/KcXKPAAAAAElFTkSuQmCC&label=lovely%20crafted%20in&message=Germany&colorA=4c566a&colorB=88c0d0"/>
  </a>
</p>


CHANGELOG.md
<p align="center">
  <a href="https://www.nordtheme.com" target="_blank">
    <picture>
      <source srcset="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/logos/heroes/logo-typography/dark/frostic/nord3/spaced.svg?sanitize=true" width="100%" media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)" />
      <source srcset="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/logos/heroes/logo-typography/light/frostic/nord6/spaced.svg?sanitize=true" width="100%" media="(prefers-color-scheme: dark)" />
      <img src="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/logos/heroes/logo-typography/dark/frostic/nord3/spaced.svg?sanitize=true" width="100%" />
    </picture>
  </a>
</p>

<p align="center"><img src="https://assets-cdn.github.com/favicon.ico" width=24 height=24/> <a href="https://github.com/arcticicestudio/nord-java/releases/latest"><img src="https://img.shields.io/github/release/arcticicestudio/nord-java.svg"/></a> <a href="https://github.com/arcticicestudio/nord-java/releases/latest"><img src="https://img.shields.io/badge/pre--release---_-blue.svg"/></a> <a href="https://github.com/arcticicestudio/nord/releases/tag/v0.2.0"><img src="https://img.shields.io/badge/Nord-0.2.0-blue.svg"/></a> <img src="http://central.sonatype.org/favicon.ico" width=24 height=24/> <a href="http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.arcticicestudio%22%20AND%20a%3A%22nord-java%22"><img src="https://img.shields.io/maven-central/v/com.arcticicestudio/nord-java.svg"/></a> <img src="https://oss.sonatype.org/favicon.ico"/> <a href="https://oss.sonatype.org/content/repositories/snapshots/com/arcticicestudio/nord-java"><img src="https://img.shields.io/badge/snapshot-0.2.0--SNAPSHOT-blue.svg"/></a> <img src="https://bintray.com/favicon.ico" width=24 height=24/> <a href='https://bintray.com/arcticicestudio/Nord/nord-java/_latestVersion'><img src='https://api.bintray.com/packages/arcticicestudio/Nord/nord-java/images/download.svg'></a> <img src="https://oss.jfrog.org/webapp/images/favicon.40285.ico" width=24 height=24/> <a href="https://oss.jfrog.org/webapp/#/artifacts/browse/tree/General/oss-snapshot-local/com/arcticicestudio/nord-java"><img src="https://img.shields.io/badge/artifactory-0.2.0--SNAPSHOT-green.svg"/></a></p>

# 0.2.0 (2016-12-03)

## Improvements

### Public API

The public API method `getVersion() : String` is now static and returns the `String` constant `VERSION` to adapt to the SonarQube minor rule [squid:S3400](https://sonarqube.com/coding_rules#q=squid%3AS3400).
The associated unit test has been refactored to test against the [ArcVer](https://github.com/arcticicestudio/arcver) library [arcver-java](https://github.com/arcticicestudio/arcver). (@arcticicestudio, #8, 7efc44cc)

The attribute

```java
private final Color COLOR;
```

conflicted with the SonarQube rule [squid:S00116](https://sonarqube.com/coding_rules#q=squid%3AS00116) and has been adapted to the Arctic Ice Studio Java code style. (@arcticicestudio, #9, 8eb03e63)

### Toolbox

Fixed the deprecated POM variable `${inceptionYear}` and replaced it with `${project.inceptionYear}`. (@arcticicestudio, #3, 1a02e6d2)

Added POM profiles for [JFrog Bintray](https://bintray.com) to deploy Maven artifacts to the [jCenter](https://bintray.com/bintray/jcenter) and snapshots to the [OSS JFrog](https://oss.jfrog.org) repository. (@arcticicestudio, #4, f74c7a57)

### Public API Documentation

Adapted the documentation style of the Nord color palette project to the JavaDoc. (@arcticicestudio, #7, 9eab5800)

## Bug Fixes

### Public API Documentation

Fixed malformed JavaDoc to prevent compilation errors caused by the new JavaDoc linter rules introduced JDK 8u92 by adding the `summary` attribute to `table` tags. (@arcticicestudio, #3, 1a02e6d2)

## Tasks

### Toolbox

Created a `circle.yml` configuration file to add [Circle CI](https://circleci.com) build integration. (@arcticicestudio, #6, f241e8de)

Added the `maven-assembly-plugin` as `assemble` POM profile (@arcticicestudio, #5, 9f6e438d)

# 0.1.0 (2016-09-26)

## Features

Implemented the public API (@arcticicestudio, #1, 8949a9baa7adf73be66b7ac8e15dfa2b57b0ac92)  
A basic usage guide for the public API can be found in the [README](https://github.com/arcticicestudio/nord-java/blob/develop/README.md#usage-guide) section.

### API

The entrypoint is the `com.arcticicestudio.nord.Nord` enum.  
Every enum constant represents a color as `Color` object.

The API provides static methods to get the HEX and RGB color code string as well as the `Color` object itself.

| Package                    | Entity | Modifier | Method                  | Description                                 |
| -------------------------- | ------ | -------- | ----------------------- | ------------------------------------------- |
| `com.arcticicestudio.nord` | `Nord` | static   | `+ hex(Color) : String` | Converts the `color` to the HEX identifier. |
| `com.arcticicestudio.nord` | `Nord` | static   | `+ rgb(Color) : String` | Converts the `color` to the RGB identifier. |
| `com.arcticicestudio.nord` | `Nord` | -        | `+ get() : Color`       | Returns the color object.                   |

<p align="center">
  <picture>
    <source srcset="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/elements/separators/iceberg/footer/dark/spaced.svg?sanitize=true" width="100%" media="(prefers-color-scheme: light), (prefers-color-scheme: no-preference)" />
    <source srcset="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/elements/separators/iceberg/footer/light/spaced.svg?sanitize=true" width="100%" media="(prefers-color-scheme: dark)" />
    <img src="https://raw.githubusercontent.com/nordtheme/assets/main/static/images/elements/separators/iceberg/footer/dark/spaced.svg?sanitize=true" width="100%" />
  </picture>
</p>

<p align="center">

src/test/java/com/arcticicestudio/nord/NordTest.java
/*
 * Copyright (c) 2016-present Sven Greb <development@svengreb.de>
 * This source code is licensed under the MIT license found in the license file.
 */

/*
 * Nord API unit tests.
 */

package com.nordtheme.nord;

import org.junit.Test;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertTrue;

/**
 * Unit tests for the {@link Nord} public API.
 *
 * @author Sven Greb &lt;development@svengreb.de&gt;
 * @since 0.1.0
 */
public class NordTest {

  @Test
  public void shouldReturnHEXColorCode() {
    assertEquals(Nord.hex(Nord.NORD0.get()), "#2E3440");
    assertEquals(Nord.hex(Nord.NORD1.get()), "#3B4252");
    assertEquals(Nord.hex(Nord.NORD2.get()), "#434C5E");
    assertEquals(Nord.hex(Nord.NORD3.get()), "#4C566A");
    assertEquals(Nord.hex(Nord.NORD4.get()), "#D8DEE9");
    assertEquals(Nord.hex(Nord.NORD5.get()), "#E5E9F0");
    assertEquals(Nord.hex(Nord.NORD6.get()), "#ECEFF4");
    assertEquals(Nord.hex(Nord.NORD7.get()), "#8FBCBB");
    assertEquals(Nord.hex(Nord.NORD8.get()), "#88C0D0");
    assertEquals(Nord.hex(Nord.NORD9.get()), "#81A1C1");
    assertEquals(Nord.hex(Nord.NORD10.get()), "#5E81AC");
    assertEquals(Nord.hex(Nord.NORD11.get()), "#BF616A");
    assertEquals(Nord.hex(Nord.NORD12.get()), "#D08770");
    assertEquals(Nord.hex(Nord.NORD13.get()), "#EBCB8B");
    assertEquals(Nord.hex(Nord.NORD14.get()), "#A3BE8C");
    assertEquals(Nord.hex(Nord.NORD15.get()), "#B48EAD");
  }

  @Test
  public void shouldReturnTheCorrectRGBColorCode() {
    assertEquals(Nord.rgb(Nord.NORD0.get()), "rgb(46, 52, 64)");
    assertEquals(Nord.rgb(Nord.NORD1.get()), "rgb(59, 66, 82)");
    assertEquals(Nord.rgb(Nord.NORD2.get()), "rgb(67, 76, 94)");
    assertEquals(Nord.rgb(Nord.NORD3.get()), "rgb(76, 86, 106)");
    assertEquals(Nord.rgb(Nord.NORD4.get()), "rgb(216, 222, 233)");
    assertEquals(Nord.rgb(Nord.NORD5.get()), "rgb(229, 233, 240)");
    assertEquals(Nord.rgb(Nord.NORD6.get()), "rgb(236, 239, 244)");
    assertEquals(Nord.rgb(Nord.NORD7.get()), "rgb(143, 188, 187)");
    assertEquals(Nord.rgb(Nord.NORD8.get()), "rgb(136, 192, 208)");
    assertEquals(Nord.rgb(Nord.NORD9.get()), "rgb(129, 161, 193)");
    assertEquals(Nord.rgb(Nord.NORD10.get()), "rgb(94, 129, 172)");
    assertEquals(Nord.rgb(Nord.NORD11.get()), "rgb(191, 97, 106)");
    assertEquals(Nord.rgb(Nord.NORD12.get()), "rgb(208, 135, 112)");
    assertEquals(Nord.rgb(Nord.NORD13.get()), "rgb(235, 203, 139)");
    assertEquals(Nord.rgb(Nord.NORD14.get()), "rgb(163, 190, 140)");
    assertEquals(Nord.rgb(Nord.NORD15.get()), "rgb(180, 142, 173)");
  }
}


src/main/java/com/nordtheme/nord/Nord.java
/*
 * Copyright (c) 2016-present Sven Greb <development@svengreb.de>
 * This source code is licensed under the MIT license found in the license file.
 */

/*
 * Nord API.
 */

package com.nordtheme.nord;

import javafx.scene.paint.Color;

import java.util.regex.Pattern;

/**
 * Official Java implementation of Nord — An arctic, north-bluish color palette.
 * Created for the clean and uncluttered design pattern to achieve a optimal focus and readability for code syntax highlighting and UI components.
 * Nord colors are numbered from {@code nord0} to {@code nord15} where each component contains a different amount of colors.
 * <p>
 *   Public API entry point of the <a href="https://github.com/nordtheme/java">Nord Java</a> project, which
 *   implements the <a href="https://github.com/nordtheme/nord">Nord</a> color palette.
 *
 * <div>
 *   <table summary="Nord Color Palette">
 *     <tbody>
 *       <tr>
 *         <th style="border:0;">Polar Night</th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(46,52,64);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(59,66,82);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(67,76,94);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(76,86,106);margin:0"></th>
 *       </tr>
 *       <tr>
 *         <th style="border:0;">Snow Storm</th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(216,222,233);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(229,233,240);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(236,239,244);margin:0"></th>
 *       </tr>
 *       <tr>
 *         <th style="border:0;">Frost</th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(143,188,187);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(136,192,208);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(129,161,193);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(94,129,172);margin:0"></th>
 *       </tr>
 *       <tr>
 *         <th style="border:0;">Aurora</th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(191,97,106);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(208,135,112);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(235,203,139);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(163,190,140);margin:0"></th>
 *         <th style="border:0;width:25px;height:30px;background-color:rgb(180,142,173);margin:0"></th>
 *       </tr>
 *     </tbody>
 *   </table>
 * </div>
 *
 * @author Sven Greb &lt;development@svengreb.de&gt;
 * @version 0.2.0
 * @since 0.1.0
 */
public enum Nord {
  /**
   * Base component color "{@code nord0}" of "Polar Night" with a RGB value of {@code rgb(46, 52, 64)} and a HEX value
   * of {@code #2E3440}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(46,52,64);margin:0"></div>
   */
  NORD0(46, 52, 64),

  /**
   * Brighter color "{@code nord1}" of the "Polar Night" component base color {@link #NORD0} with a RGB value of
   * {@code rgb(59, 66, 82)} and a HEX value of {@code #3B4252}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(59,66,82);margin:0"></div>
   */
  NORD1(59, 66, 82),

  /**
   * Brighter color "{@code nord2}" of the "Polar Night" component base color {@link #NORD0} with a RGB value of
   * {@code rgb(67, 76, 94)} and a HEX value of {@code #434C5E}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(67,76,94);margin:0"></div>
   */
  NORD2(67, 76, 94),

  /**
   * Brighter color "{@code nord3}" of the "Polar Night" component base color {@link #NORD0} with a RGB value of
   * {@code rgb(76, 86, 106)} and a HEX value of {@code #4C566A}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(76,86,106);margin:0"></div>
   */
  NORD3(76, 86, 106),

  /**
   * Base component color "{@code nord4}" of "Snow Storm" with a RGB value of {@code rgb(216, 222, 233)} and a HEX value
   * of {@code #D8DEE9}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(216,222,233);margin:0"></div>
   */
  NORD4(216, 222, 233),

  /**
   * Brighter color "{@code nord5}" of the "Snow Storm" component base color {@link #NORD4} with a RGB value of
   * {@code rgb(229, 233, 240)} and a HEX value of {@code #E5E9F0}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(229,233,240);margin:0"></div>
   */
  NORD5(229, 233, 240),

  /**
   * Brighter color "{@code nord6}" of the "Snow Storm" component base color {@link #NORD4} with a RGB value of
   * {@code rgb(236, 239, 244)} and a HEX value of {@code #ECEFF4}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(236,239,244);margin:0"></div>
   */
  NORD6(236, 239, 244),

  /**
   * Bluish core color "{@code nord7}" of the "Frost" component with a RGB value of {@code rgb(143, 188, 187)} and a
   * HEX value of {@code #8FBCBB}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(143,188,187);margin:0"></div>
   */
  NORD7(143, 188, 187),

  /**
   * Bluish core accent color "{@code nord8}" of the "Frost" component with a RGB value of {@code rgb(136, 192, 208)}
   * and a HEX value of {@code #88C0D0}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(136,192,208);margin:0"></div>
   */
  NORD8(136, 192, 208),

  /**
   * Bluish core color "{@code nord9}" of the "Frost" component with a RGB value of {@code rgb(129, 161, 193)} and a
   * HEX value of {@code #81A1C1}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(129,161,193);margin:0"></div>
   */
  NORD9(129, 161, 193),

  /**
   * Bluish core color "{@code nord10}" of the "Frost" component with a RGB value of {@code rgb(94, 129, 172)} and a
   * HEX value of {@code #5E81AC}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(94,129,172);margin:0"></div>
   */
  NORD10(94, 129, 172),

  /**
   * Colorful color "{@code nord11}" of the "Aurora" component with a RGB value of {@code rgb(191, 97, 106)} and a
   * HEX value of {@code #BF616A}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(191,97,106);margin:0"></div>
   */
  NORD11(191, 97, 106),

  /**
   * Colorful color "{@code nord12}" of the "Aurora" component with a RGB value of {@code rgb(208, 135, 112)} and a
   * HEX value of {@code #D08770}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(208,135,112);margin:0"></div>
   */
  NORD12(208, 135, 112),

  /**
   * Colorful color "{@code nord13}" of the "Aurora" component with a RGB value of {@code rgb(235, 203, 139)} and a
   * HEX value of {@code #EBCB8B}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(235,203,139);margin:0"></div>
   */
  NORD13(235, 203, 139),

  /**
   * The color {@code nord14} of the "Aurora" component with an RGB value of {@code rgb(163, 190, 140)} and an HEX
   * value of {@code #A3BE8C}.
   * <div style="border:none;width:25px;height:30px;background-color:rgb(163,190,140);margin: 0"></div>
   */
  NORD14(163, 190, 140),

  /**
   * Colorful color "{@code nord15}" of the "Aurora" component with a RGB value of {@code rgb(180, 142, 173)} and a
   * HEX value of {@code #B48EAD}.
   * <p>
   *   <div style="border:none;width:25px;height:25px;background-color:rgb(180,142,173);margin:0"></div>
   */
  NORD15(180, 142, 173);

  /**
   * Prefix pattern to convert RGB- and HEX color values.
   */
  private static final Pattern COLOR_PREFIX = Pattern.compile("0x", Pattern.LITERAL);

  /**
   * The RGB color code.
   */
  private final Color color;

  /**
   * The <a href="https://semver.org">SemVer</a> version of the public API.
   */
  private static final String VERSION = "0.2.0";

  /**
   * Constructs a new color object.
   *
   * @param red the value for the additive primary color {@code red}
   * @param green the value for the additive primary color {@code green}
   * @param blue the value for the additive primary color {@code blue}
   */
  Nord(final int red, final int green, final int blue) {
    color = Color.rgb(red, green, blue);
  }

  /**
   * Converts the specified color to the HEX identifier.
   *
   * @param color the color object to convert
   * @return the string representation of the HEX identifier
   */
  public static String hex(final Color color) {
    return COLOR_PREFIX.matcher(color.toString()).replaceAll("#").substring(0, 7).toUpperCase();
  }

  /**
   * Converts the specified color to the RGB identifier.
   *
   * @param color the color object to convert
   * @return the string representation of the RGB identifier
   */
  public static String rgb(final Color color) {
    String hex = COLOR_PREFIX.matcher(color.toString()).replaceAll("");
    String hexRed = hex.substring(0, 2).toUpperCase();
    String hexGreen = hex.substring(2, 4).toUpperCase();
    String hexBlue = hex.substring(4, 6).toUpperCase();

    String intRed = Integer.toString(Integer.parseInt(hexRed, 16));
    String intGreen = Integer.toString(Integer.parseInt(hexGreen, 16));
    String intBlue = Integer.toString(Integer.parseInt(hexBlue, 16));

    return String.join("", "rgb(", intRed, ", ", intGreen, ", ", intBlue, ")");
  }

  /**
   * The color constant as color object.
   *
   * @return the color object
   */
  public Color get() {
    return color;
  }

  /**
   * Returns the <a href="https://semver.org">SemVer</a> version of the public API.
   *
   * @return the SemVer version string
   * @see <a href="https://semver.org">SemVer</a>
   */
  public static String getVersion() {
    return VERSION;
  }
}
